liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/RunServerMojo.java

```diff
@@ -44,12 +44,17 @@ protected void doExecute() throws Exception {
         if (skip) {
             return;
         }
-        if (isInstall) {
-            installServerAssembly();
-        } else {
-            log.info(MessageFormat.format(messages.getString("info.install.type.preexisting"), ""));
-            checkServerHomeExists();
+        
```

Comment:
Will the call to runLibertyMojoCreate() below take care of checking for installServerAssembly and checkServerHomeExists?